lib/buildingsync/version.rb

```diff
@@ -41,5 +41,5 @@
 # provides features for reading BldgSync XML files, generating baseline models, creating scenario workflows, running simulations and adding simulation results to the BldgSync file
 module BuildingSync
   # version of the BuildingSync gem
-  VERSION = '0.2.1'
+  VERSION = '0.2.2'
```

Comment:
I think we should bump this to 0.3.0? thoughts on why not to?

Reply:
I think that makes sense, this is a major change.  It shouldn't break anything.